content/release-10-17-0/devices-10-17-0.md

```diff
@@ -25,7 +25,7 @@ Fix Version</th>
 <tr>
 <td>Device Management</td>
```

Comment:
The ticket had already been added, but there was a copy/paste error with the ticket number, so you couldn´t find it. In general, you do not need to add improvements and fixes via PR, since I take care of it via the entries in the Jira tickets.